pkg/apis/app/v1beta1/application.go

```diff
@@ -89,7 +109,7 @@ func (a *Application) Observables(scheme *runtime.Scheme, rsrc interface{}, rsrc
 				observables = append(observables, observable)
 			}
 		}
-
+		log.Printf("\n")
```

Comment:
Worth logging the error (if non-nil) returned from `scheme.New(gvk)`

Reply:
done.